### DISCUSSION TITLE
Crash Flight-Tracker-Plugin and Prepar3D Version 6

### DISCUSSION THREAD

**Discussion by fsbloch** [nguyenquyhy/Flight-Tracker-StreamDeck#399] [General]
If I use the Flight-Tracker-Plugin with my Elgato Stream Deck an Prepar3D Version 6 I get an crash to desktop. If the Flight-Trakcer-Plugin ist disabled, I get no crash. What could be the cause and how can the fault be rectified?

![2023-07-15 11_41_16-Ereigniseigenschaften - Ereignis 1000,  NET Runtime](https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/assets/38840762/b36451ed-9fea-4301-a7a9-daa4667cd0d4)

**Comment by nguyenquyhy** [comment]
On the plugin, please try to update to version 0.50 and see if the plugin stops crashing.

I can't fix crashes in P3D though, Lookheed Martin will have to fix that, especially if it happens only on v6.

**Comment by fsbloch** [comment]
The new version 0.51 of the plugin stops crashing :-).
